.ci_support/linux_.yaml

```diff
@@ -1,6 +1,6 @@
 channel_sources:
 - conda-forge,defaults
 channel_targets:
-- conda-forge main
+- conda-forge jupyterlab_rc
```

Comment:
I think we shouldn't be touching this file?

Reply:
Note in the [instructions](https://github.com/conda-forge/cfep/blob/master/cfep-05.md#specification):

> NOTE: A rerender needs to happen for these changes to reflect in CI files.

So we shouldn't touch these files, but instead ask for a rerender.